### DISCUSSION TITLE
Unable to pay for the subsribed plan

### DISCUSSION THREAD

**Discussion by y3owk1n** [swellstores/community#112] [Help] [answered]
Hi Swell Team, I've been trying with 3 different credit card but seems like it can never get charged for whatever reason. Would like to get some help, as I would not want the service to get interrupted.

![Screenshot 2023-01-26 at 3 37 36 PM](https://user-images.githubusercontent.com/62775956/214781475-7c63b896-3366-47ca-a693-45f10209cb63.png)

**Comment by steven-k1** [comment, marked as answer]
Hello @y3owk1n, I looked over your account, and from what I can see, the issue was due to the default billing card not being set within our admin. I checked your account dashboard and saw the card currently set within there and put it as the default card in our admin.

However, I tried to process the payments for the invoices, and the card was declined.

I'll reach out to you directly through the email you have on file for your account to confirm the card you'd like to process the payment with.
